src/XML/src/FMI3/fmi3_xml_model_description.c

```diff
@@ -843,12 +833,8 @@ fmi3_xml_variable_t* fmi3_xml_get_variable_by_vr(fmi3_xml_model_description_t* m
     fmi3_xml_variable_t *pkey = &key;
     void** found;
 
-    if (!md->variablesByVR) return NULL;
-
-    // TODO: When variablesByVR no longer contains alias variables, then just return by index.
```

Comment:
This comment was false. You can't just do that (since VRs don't have to match the variable index)